packages/analytics-types/src/element-interactions.ts

```diff
@@ -0,0 +1,78 @@
+import { Logger } from './logger';
+
+export type ActionType = 'click' | 'change';
+
+/**
+ * Default CSS selectors to define which elements on the page to track.
+ * Extend this list to include additional elements to track. For example:
+ * ```
+ * autocapturePlugin({
+ *    cssSelectorAllowlist: [...DEFAULT_CSS_SELECTOR_ALLOWLIST, ".my-class"],
+ * })
+ * ```
+ */
+export const DEFAULT_CSS_SELECTOR_ALLOWLIST = [
+  'a',
+  'button',
+  'input',
+  'select',
+  'textarea',
+  'label',
+  '[data-amp-default-track]',
+  '.amp-default-track',
+];
+
+/**
+ * Default prefix to allo the plugin to capture data attributes as an event property.
+ */
+export const DEFAULT_DATA_ATTRIBUTE_PREFIX = 'data-amp-track-';
+
+export interface ElementInteractionsOptions {
```

Comment:
Same with this interface

Reply:
Recently we had to add one additional property inside this called `actionClickAllowlist`, I'm curious if anything needs to change other than this to support it?

Reply:
Once this version is released, we should import these `const` and `interface` from `analytics-types` in `plugin-autocapture-browser`. I copied these from v0.9.2, so we need to all changes after it